### PR TITLE
Add kfilt to helm-gke-setup action

### PR DESCRIPTION
**Pull request by yannick-roeder**

### CODE REVIEW

File: actions/helm-gke-setup/action.yaml [hunk 1]
```diff
@@ -53,6 +53,12 @@ runs:
       with:
         version: ${{ inputs.kubectl-version }}
 
+    - name: Install kfilt
+      shell: bash
+      run: |
+        curl -L "https://github.com/ryane/kfilt/releases/download/v0.0.7/kfilt_0.0.7_linux_amd64" -o /usr/local/bin/kfilt
```
Comment (philipp94831) [inline actions/helm-gke-setup/action.yaml:59]: Why not use a docker image that has kfilt available?

Comment (yannick-roeder) [reply actions/helm-gke-setup/action.yaml:59]: Yes, building a docker image would be the next step. I would prefer to move this into this issue: https://github.com/bakdata/ci-templates/issues/16 
Is that fine?

By moving it, I can first finish building the docker release ci template (see this branch: https://github.com/bakdata/ci-templates/blob/feature/docker-release-workflow/.github/workflows/docker-release.yaml) and build out proper ci-images here: https://github.com/bakdata/ci-images/tree/main

Comment (philipp94831) [reply actions/helm-gke-setup/action.yaml:59]: Please update the issue and leave a not on what to install. Also leave a TODO here to remove the installation

**Review by philipp94831 — APPROVED** [review]

File: actions/helm-gke-setup/action.yaml (hunk 1 above)
Comment (yannick-roeder) [reply actions/helm-gke-setup/action.yaml:60]: Done

**Pull request merged**